contracts/lib/time_series/convert.cairo

```diff
@@ -0,0 +1,90 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.pow import pow
+from starkware.cairo.common.math_cmp import is_le
+from starkware.cairo.common.math import unsigned_div_rem, assert_not_zero
+
+func convert_via_usd{range_check_ptr}(a_price_in_usd, b_price_in_usd, b_decimals) -> felt {
```

Comment:
Could rename `b_decimals` to `output_decimals`, confused me for a while why we needed `b_decimals` but not `a_decimals` until I realized it is because a and b are both /usd so have the same number of decimals